tests/helpers/docker/tendermint.py

```diff
@@ -282,6 +282,42 @@ def _create_one(self, i: int) -> Container:
         container = self._client.containers.run(**run_kwargs)
         return container
 
+    def _fix_persistent_peers(self) -> None:
+        """
+        Fix the persistent peers' ports in the configuration file.
+
+        Since we are running all the ABCIs at the same host for our e2e tests, we shift the ports by 10 for each
+        added node. Therefore, we need to override the default persistent peers in the config files,
+        in order for them to use the correct ports.
+        """
+        nodes_config_files = list(Path().cwd().glob("**/config.toml"))
+        assert (
+            nodes_config_files != []
+        ), "Could not detect any config files for the nodes!"
```

Comment:
Adding this part, we can now fail early if the config files are not detected in order to make sure that we always override the persistent peers with the correct configuration.

The e2e tests will currently fail here. By removing the following line, we manage to detect the config files:

https://github.com/valory-xyz/open-autonomy/blob/5886e661dcec862d3ee688343b5f1d8f4bcf9878/tests/test_agents/base.py#L119

However, the execution fails later in line 319, because we do not have permission to write to the file:

https://github.com/valory-xyz/open-autonomy/blob/5886e661dcec862d3ee688343b5f1d8f4bcf9878/tests/helpers/docker/tendermint.py#L319